services/app-web/src/components/SubmissionSummarySection/UploadedDocumentsTable/UploadedDocumentsTable.tsx

```diff
@@ -61,27 +83,23 @@ export const UploadedDocumentsTable = ({
 
         void refreshDocuments()
     }, [documents, getKey, getURL])
-    
+
+    // Empty State
+    if (refreshedDocs.length < 1) {
```

Comment:
is `.length < 1` a common js-ism? It seems more clear to me to do `.length === 0`

Reply:
no its not. I can try to write things that way, agree its clear. If I think hard maybe its  cuz I have a tendency to just avoid `=== 0` cuz of typoed `== 0` being falsey? but with linting that doesn't happen.